python/lsst/meas/extensions/ngmix/config.py

```diff
@@ -279,6 +283,12 @@ class BasicProcessConfig(ProcessCoaddsTogetherConfig):
         optional=True,
         doc='prefix to add to plot names',
     )
+    seed = Field(
```

Comment:
We don't want the seed to actually be set in the config file, since we use the same config file for all runs of this code.

Reply:
Does it not change the output? If so, then surely it needs to be in the config.

You can always use multiple configs and/or set additional fields via the command line like `seed=0`. Why wouldn't you want it to be a config setting?

Reply:
We would need a different config for every single run of the code.  On the contrary, we want a single config file for an entire  *campaign*, for example Year 5 processing.  Just a single file that describes the processing for an entire campaign.

Better to send the seed on the command line, as you say.

Reply:
That's what I meant, though - you can set (override, even) config parameters from the command line in both gen2 and 3, as in `processDeblendedCoaddsNGMixMax.py ... -c seed=1`, and then it will be persisted. If it's not a config parameter then I don't think it's persisted anywhere in either generation.

Reply:
Then I think we need it to be a required parameter to force people to set the seed